### PR TITLE
Add v9.0.2 to CHANGELOG.md

### PR DESCRIPTION
**Pull request by mborsz**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
/kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Adds an update about v9.0.2 version to CHANGELOG.md (as requested in https://github.com/kubernetes/kubernetes/pull/80630#pullrequestreview-267282804)
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```
/assign @MrHohn 

/cc @wojtek-t

### CODE REVIEW

File: cluster/addons/addon-manager/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+### Version 9.0.1  (Thu August 1 2019 Maciej Borsz <maciejborsz@google.com>
```
Comment (MrHohn) [inline cluster/addons/addon-manager/CHANGELOG.md]: nit: 9.0.2

**Review by MrHohn — APPROVED** [review]
Thanks Maciej, just one nit.

File: cluster/addons/addon-manager/CHANGELOG.md (hunk 1 above)
Comment (mborsz) [reply cluster/addons/addon-manager/CHANGELOG.md]: Thanks, fixed! :)

**Pull request merged**